content/200-orm/200-prisma-client/000-setup-and-configuration/050-databases-connections/115-connection-pool.mdx

```diff
@@ -176,7 +176,7 @@ For more details on what is available in the metrics output, see the [About metr
 
 #### Default pool timeout
 
-The default connection pool timeout is 10 seconds. If the query engine does not process a particular query within that time, it throws an exception and moves on to the next query in the queue.
+The default connection pool timeout is 10 seconds. If the query engine does not get a connection from the connection pool within that time, it throws an exception and moves on to the next query in the queue.
```

Comment:
```suggestion
The default connection pool timeout is 10 seconds. If the Query Engine does not get a connection from the database connection pool within that time, it throws an exception and moves on to the next query in the queue.
```
I presume it's from the database's connection pool, correct? If not, feel free to ignore my suggestion. 😄